core/src/workflows/get_entry_result.rs

```diff
@@ -68,22 +86,30 @@ pub async fn get_entry_result_workflow<'a>(
                 context
                     .state()
                     .expect("state uninitialized! :)")
-                    .get_headers(entry_with_meta.entry.address().clone())?
+                    .get_headers(entry_with_meta.entry_with_meta.entry.address().clone())?
```

Comment:
Why is this still here? Shouldn't this be replaced by the code that was added?

Reply:
In line 68 above, `get_entry_with_meta_workflow` now returns an `EntryWithMetaAndHeader`, so the header is already there..

Maybe that function and the local variable `maybe_entry_with_meta` should be renamed as well to reflect that the header is already included?

Reply:
So I think this would even overwrite the right header that was returned by the new code with what is found locally, right? So I think this is a bug and needs changing.

Reply:
Yes,  I think I missed this during my changes. This is a bug

Reply:
I think this might have been remnants of the old code